README.md

```diff
@@ -15,6 +15,8 @@ The common interface mentioned above that all chaos extensions must implement is
 
 ## How to use
 
+> Note: node version `>= 4.9.x` is __required__.
```

Comment:
can this go as a badge at the top as well?

Reply:
potentially: 
![node](https://img.shields.io/node/v/platform-chaos.svg)